src/components/publishing/image_collection_item.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react"
+import styled from "styled-components"
+
+interface FillwidthItemProps extends React.HTMLProps<HTMLDivElement> {
+  width?: number
+  height?: number
+  margin?: number
+  key: string
+}
+
+const FillwidthItem: React.SFC<FillwidthItemProps> = props => {
+  const { key, ...rest } = Object.assign({}, props)
+  return <div {...rest} key={key} />
```

Comment:
Btw, if `props.margin` is not defined in `props.margin + "px" || 0`, the result would be `"undefinedpx"`, not `0`. I assume that’s not the intention, perhaps `props.margin && props.margin + "px" || 0` is?